### PR TITLE
[pt-br] Add docs/reference/glossary/manifest.md

### PR DESCRIPTION
**Pull request by julialamenza**

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/manifest.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Manifesto
+id: manifest
+date: 2019-06-28
+short_description: >
+  Uma especificação serializada de um ou mais objetos da API Kubernetes.
```
Comment (edsoncelio) [inline content/pt-br/docs/reference/glossary/manifest.md:6]: "API Kubernetes" can be "API do Kubernetes"

File: content/pt-br/docs/reference/glossary/manifest.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+aka:
+tags:
+- fundamental
+---
+ Especificação de um objeto da API Kubernetes em formato JSON ou YAML.
```
Comment (edsoncelio) [inline content/pt-br/docs/reference/glossary/manifest.md:11]: "API Kubernetes" can be "API do Kubernetes"

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hey @julialamenza!
Thanks for the contribution! 
Just a few (small) changes :)

**Comment by edsoncelio** [comment]
Thanks @julialamenza!
/lgtm
PTAL @stormqueen1990

**Review by stormqueen1990 — COMMENTED** [review]
Thanks for your contribution, @julialamenza! 🎉

/approve

**Pull request merged**